cmake/mysql_version.cmake

```diff
@@ -38,11 +38,7 @@ CONFIGURE_FILE(
 
 MACRO(MYSQL_GET_CONFIG_VALUE keyword var file)
  IF(NOT ${var})
-<<<<<<< HEAD
    FILE (STRINGS ${CMAKE_SOURCE_DIR}/${file} str REGEX "^[ ]*${keyword}=")
```

Comment:
shouldn't u use MYSQL_VERSION? btw, are we making the PXB version bump to server version in this release?

Reply:
${file} we were using in the previous release instead of using VERSION file. yes, we are bumping PXB version to use 8.0.22 instead of 8.0.15.

Reply:
The conflict resolution shows that you still use {file}. Shouldn't you use `${CMAKE_SOURCE_DIR}/MYSQL_VERSION}` instead of `${file}`

Reply:
so ${file} is either XB_VERSION or MYSQL_VERSION

Reply:
ok, lets leave it as is then